components/_Progress-Bar/progress-bar-initialization.js

```diff
@@ -0,0 +1,111 @@
+        progressBarFeatures2.init("ProgressBarFeatures2", 5, 0); //Initialize the progress bar with 5 steps.
+    }
+
+    const progressBarFeatures3El = document.getElementById("ProgressBarFeatures3");
+    if(progressBarFeatures3El) {
+        //Set the progress bar to a specific step.
+        const progressBarFeatures3 = Object.create(window.msdsProgressBar);
+        progressBarFeatures3.init("ProgressBarFeatures3", 5, 3); //Initialize the progress bar with 5 steps.
+    }
+
+    //Progress bar Size scripts------------------
+    const progressBarSize1El = document.getElementById('ProgressBarSize1');
+    if(progressBarSize1El) {
+        const progressBarSize1 = Object.create(window.msdsProgressBar);
+        progressBarSize1.init('ProgressBarSize1', 5, 2); //Initialize the progress bar with 5 steps.
+    }    
+            
+    const progressBarSize2El = document.getElementById('ProgressBarSize2');
+    if(progressBarSize2El) {
+        //Set the progress bar to a specific step by default.
+        const progressBarSize2 = Object.create(window.msdsProgressBar);      
+        progressBarSize2.init('ProgressBarSize2', 5, 3); //Initialize the progress bar with 5 steps.
+    }
+     
+    
+    //Light Theme scripts------------------       
+    //Initialize the progress bar with 12 steps.        
+    const lightThemeProgressBarEl = document.getElementById("LightThemeProgressBar");
+    if(lightThemeProgressBarEl) {
+        const lightThemeProgressBar = Object.create(window.msdsProgressBar);
+
+        let lightThemeBaseNumberElement = document.querySelectorAll('#LightThemeBaseNumber')[0]    
+        lightThemeBaseNumberElement.value = 12;
+
+        lightThemeProgressBar.init("LightThemeProgressBar", lightThemeBaseNumberElement.value, 0);  
+        
+        document.querySelector('#LightThemePrevStep').addEventListener('click', event => {            
+            lightThemeProgressBar.prev();
+        })
+    
+        document.querySelector('#LightThemeNextStep').addEventListener('click', event => {
+            lightThemeProgressBar.next();
+        })
+    
+        document.querySelector('#LightThemeBaseNumber').addEventListener('change', event => {
+            lightThemeProgressBar.init("LightThemeProgressBar", event.target.value, 0);        
+        })    
+    
+        const lightThemeStartFromOneElement = document.querySelector('#LightThemeStartFromOne');
+        lightThemeStartFromOneElement.addEventListener('change', event => {        
+            lightThemeProgressBar.startFromOne = lightThemeStartFromOneElement.checked;
+            const baseNumber = document.querySelectorAll('#LightThemeBaseNumber')[0].value;
+            lightThemeProgressBar.init("LightThemeProgressBar", baseNumber, 0);
+        })
+    }               
+          
+    
+    //Dark Theme scripts------------------        
+    //Initialize the progress bar with 12 steps.
```

Comment:
This method seems to be doing the exact same thing as the one above it? the same logic is applied. this should probably be refactored into one simple method.